app/services/sufia/user_stat_importer.rb

```diff
@@ -5,7 +5,11 @@ class UserStatImporter
     UserRecord = Struct.new("UserRecord", :id, :user_key, :last_stats_update)
 
     def initialize(options = {})
-      @verbose = options[:verbose]
+      if options[:verbose]
```

Comment:
This doesn't strike me as appropriate for `UserStatImporter` (especially extending `Rails.logger` for everyone..). I'd feel better if it were pushed to the rake task, but maybe there's an even better way (say, `UserStatImporter#logger` that can switch between a stdout logger and the ordinary rails logger?)